### DISCUSSION TITLE
大學一學期16周還是18周

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#992] [Q&A]
最近看到台科大還有台大都開始放暑假了，羨慕之餘也不禁開始想，16周上課真的好嗎，據我在台科大問到的同學，他們覺得好像什麼都沒學到就寒假了，還度如18周至少能學到東西，像我們系有些課就是16周有些課就是18周，雖然我本人是覺得沒差拉，但是好像有挺多人有反感的，各位覺得一學期上課16周好科還是18周?
歡迎留言~~

**Comment by ruthchen1020** [comment]
我希望像現在這樣，課很多很重的就18周，少的就16周，這樣才不會全部卡在最後一周，最後一周也可以比較用心且完整的教完和讀完。

**Comment by HuangHsinTzu** [comment]
我也偏向現在這樣!
有些比較重的課程就是沒辦法在16周內上完
如果硬要在16周內上完課程，老師就會需要趕課，同學就會學的比較不扎實
但的確有些課程是可以在16周內上完的
所以我認為16+2是最適當的!

**Comment by peggy0329** [comment]
我也支持目前這樣的做法
後面兩周老師可以彈性安排，這樣期末考跟報告也不會全部卡在一起
學生也比較彈性安排時間

**Comment by ru0372572** [comment]
我覺得彈性安排就會變成老師可以選擇不管這兩堂課，認真或上不完的確是可以用來調整時間，但如果什麼都沒做感覺學費好像被偷了。

**Comment by BlahFish1006** [comment]
我也偏向現在這樣~! 老師可以自己安排進度和時間調整，
通識課和系上主科的考試也能分開來考，對我來說學習效果和考試的準備都比較好。

**Comment by delia07152** [comment]
我覺得主科十八周很剛好诶，至於通識課那些十六周就夠了哈哈。

**Comment by ching020721** [comment]
我偏向18周，這樣更能每一科都有學好學滿，不然16周不管期中還是期末都太倉促的感覺，第8-10周和第16-18周都在為考試焦慮，反而課餘活動都沒辦法放下心好好準備或放鬆。

**Comment by Sylvie0601** [comment]
我也喜歡像現在這樣，我覺得像是系上主科就可以到18周，而一般的副科、通識課這種的感覺16周就可以結束，因為主科16周就結束的話感覺就更學不到甚麼東西了，畢竟18周要上完課有時候已經夠趕了，而且現在這樣的話剩下兩周可以全心衝刺主科我覺得超級棒!

**Comment by Kotori0501** [comment]
我比較喜歡現在這樣，
因為期末考相對變得比較分散，
這樣比較好分配讀書時間，
如果像之前那樣全部都集中在同一週考，
對於經常臨時抱佛腳的我，
真的非常難全部讀完

**Comment by Valentinaa0214** [comment]
我還是想回到一學期十八周的日子🥲
十六周大幅壓縮了授課品質和內容，教授教得很趕，
我也學得很痛苦，感覺學的不夠深入、一知半解，
期中考完馬上就接著期末，沒什麼喘息的時間，
也想多點時間跟同學相處。

**Comment by moonnight3503** [comment]
我傾向課業較重的的18周，相較之下較輕鬆的課程16周即可
我覺得這樣期末分散一點可以準備得比較充足，
雖然現在有些課程期中期末拉得蠻進的，但我覺得可能只是有些教授還沒調適過課程安排
可能一陣子就會習慣了

**Comment by manipulateed** [comment]
可能由於16週的關係 某些內容比較多的科目老師可能在教學的進度會比較快點，導致上課的效率可能沒有那麼高，當然回去自己複習也是很重要，但我還是蠻喜歡16+2 的他讓後面到寒假前的收尾節奏很舒服

**Comment by s96015welson** [comment]
我比較喜歡16週的課程安排，這樣期末報告不會和期末考時間衝突，較好安排讀書計畫。然而這學期確實許多課程上課步調都快了很多，要提早開始做準備，按時複習課程內容，不然很快就會跟不上進度了。

**Comment by dabbie109303501** [comment]
我蠻支持16+2週的 像現在進度快 內容比較好上手的科目都已經結束了
有很多時間可以自己規劃和安排其他比較繁重的主科
這樣很好

**Comment by Guagua1234** [comment]
我也認為18週較合適，
主科真的較重，也沒辦法在16週內上完，
雖然18週也不一定上的完，
還是比一直在非上課時間加課還要好了。
寒暑假雖然很棒，也要充分學習完之後再放會比較好

**Comment by Tuoresa** [comment]
原先認為16-18週的彈性使得準備好放假的心情會有所落差，
但實際也明顯感覺到拉長的時間線能讓原本緊湊的交疊的報告與考試有更多的時間，
可以有較充裕的準備。
雖然放假時間沒有變長，但也反向帶來了一些益處。

**Comment by dinosaur1009** [comment]
我覺得統一會比較好.....不然心態蠻容易不平衡的

**Comment by Zoe06060606** [comment]
我覺得中央現在的做法其實還不錯，至少分散不少主科壓力，因為不會和通識考試時間撞在一起，然後期末前兩週也可以不用上體育課了🤣

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
就我看來16週對有些課程來說的確有點太短 再著像有些理工組 有些科目甚至要考到三次 16週真的應付不來

**Comment by Doreen331** [comment]
我個人喜歡16週，雖然期中期末全部都被塞在同一個禮拜，但其實即使是18週，多數人也是等到快要來不及了才開始抱佛腳，現在16+2可以讓各科的老師彈性調整課程內容，不過目前我個人遇到最大的問題就是每個學校的週數都不一樣，有人現在放假了有人還在期末地獄，要約人出去玩挺麻煩的

**Comment by Twenty0522** [comment]
16周真的滿趕進度的，期中結束馬上接著期末，12月活動比較多，想參加又必須顧到期末，有種大學過得很短的感覺。

**Comment by Jzd314** [comment]
16+2感覺是最彈性的，有些課程18週都上不完，何況是16週，因此留下兩週的彈性時間也讓期末考可以避免全部塞在同一星期

**Comment by tower20021202** [comment]
我覺得16+2目前還有很多進步空間，雖然這樣可以避免期末考都塞在短短兩週的期末週，但有時長痛不如短痛，四周的期末考真的是個長期折磨

**Comment by JessyCheng2003** [comment]
我也認為目前學校讓各科老師決定16或18周的作法蠻不錯的，一些loading比較少的課程就可以在16周提早結束，而剩下兩周的時間，可以專心在主科或是一些重要科目上，而且考試日期就不會很緊湊。

**Comment by Elaine2899** [comment]
我也相當認同現在的做法，雖然看到其他同學放假會覺得心裡不平衡，但我也不想這學期好像什麼都沒學到就過去了。而且這樣的做法也可以把考試時間錯開，讓我們的時間安排更有彈性，如果當學期剛好都是16周的課還會有賺到2周的小快樂。

**Comment by zephyrxin01** [comment]
我覺得16+2比較理想，因為這樣對主科來說教學品質不會受到那麼大的影響，且像是一些通識課能彈性一點。除此之外，期末週的壓力也比較不會太大，因為大多數課程都能比較分散的安排期末考或期末專題。

**Comment by harrycoconut** [comment]
我認為16週比較好，最後的兩週能當作自我探索，讓學生在學期結束前去對自己在整個學期的一個檢討，以我的觀點來看，16+2週的說法會比較正確

**Comment by ashleyy-huang** [comment]
我覺得課程內容較多、較重要的可以16+2周，但是通識課或是其他非本科系的課程就可以盡量在16周結束就好了，這樣就可以減緩學生的課業壓力，也可以利用最後兩周專心鑽研主修科目，這樣彈性的課程話規劃對於學生來說是德政

**Comment by sjh501** [comment]
我覺得現在16+2的規劃還不錯，老師可以依照課程的性質做規劃，一些負荷比較小、沒有進度壓力的課程排16周；反之一些較為困難、深入的課程，尤其是學分頗重的課，外加一到兩周的時間分散對我來說也會輕鬆不少。當然每次期末聽到某某某已經放假還是會有些羨慕，但現在這個階段為了能夠良好的學習犧牲一兩周的時間，我覺得是值得的。

**Comment by ALAN30359704** [comment]
我認為16周的安排其實讓系上原本18周的課程，因爲以16周爲原則少教了實務上的應用的部分，反而讓我覺得最重要的部分沒有完成。

**Comment by Superlan8** [comment]
其實說實話，我個人蠻喜歡16+2的彈性調整，除了避免最後一周所有考試都擠在一起外，還給了我很多學習時間分配的彈性，但內心看到同學們都已經開始放假，自己卻還要留著繼續讀考試，心裡還是會有點小羨慕。

**Comment by ziihui** [comment]
覺得16+2很不錯，不會全部的考試都同時擠在同一週，更能有效分配讀書及做報告時間！我覺得若是全部都擠在16週扣掉第一週跟期中週，感覺不能學得很紮實，然後期末會累死⋯

**Comment by Alexgns** [comment]
我認為16周較為彈性，上大學除了上課之外，主要任務為培養自主規劃能力，因此不論16周或18周，都需要自行安排學習計畫，當然上課聽講也極為重要，但不見得每位老師都能夠清楚的將觀念傳授，所以最重要的是自己規劃時間

**Comment by xiang305** [comment]
我個人認為18週比較好，有些課因為內容太多，要不停地趕進度，很多地方就講得很快，又或是要我們回去自習。雖然改成16週看似可以提早放假，但有些老師還是會在彈性運用的兩周內排課。

**Comment by quan0715** [comment]
我覺得16+2有好有壞，因為是彈性做安排，就會有很多不一樣的考試時間 16 17 18 週都要考試，有些課一學期要考3 次有些課一學期要考2次，這樣算下來整學期就會有人每個禮拜都有個兩三顆再考，蠻慘的。

**Comment by Aaron0810** [comment]
我是覺得其實都可以，但比較希望學校可以統一就好，要學人家台大學期16周就要徹底執行，不然就一樣18周就好，不然這樣分很開常常會有周邊朋友都開始在玩了結果只剩自己還有多撐一兩個禮拜，抑或是你幾乎所有課都結束了但其中剩的唯一一堂課在18周的星期五考試，這樣就很白癡，所以我只希望學校可以統一。

**Comment by brian431** [comment]
對一些教學內容很充實的課來說應該還是要十八週才夠，但以我目前在系上修的課來說。因為內容實在是沒什麼，所以我很希望十六週快速教完放我去放假。

**Comment by poweichen00** [comment]
我覺得18週比較好，16週相對來講考試壓力就會比較大，多兩週能把一些功課或考試分掉，上課進度也不會這麼趕。

**Comment by Evening218** [comment]
我覺得16+2比較好!不會讓所有期末的事情都卡在一起,唯一比較不好的就是因為我們跟老師們都還在適應期,所以老師們也常常無法確定自己甚麼時候可以結束課程,這個對假期的行程安排會比較麻煩一點,因為我根本沒辦法知道甚麼時候才會放假

**Comment by Mitochondrionjessica** [comment]
喜歡16+2，因為主科通常都會上滿18周，通識或是國文英文通常就16周，可以把16周時把非主科的報告作業考試結束，剩下兩周好好拚主科期末。

**Comment by William08290829** [comment]
我也覺得維持現在這樣還不錯，可以讓教授自己決定要16 17還是18周考，也讓學生可以有比較多的時間可以讀書，不會造成每一科全部塞在一個禮拜一次考完，那樣感覺會全部爆炸明年重來。

**Comment by tzurou** [comment]
學校實行16+2周到現在也已經有1年的時間了，我個人覺得還是維持彈性18周比較好，老師可以自行決定上課週數，我們系的老師這學期的課程大綱也都是16周，除非遇到連假或是老師突然有事才多順延一周變成17周，但時機上課週數還是維持16周，個人覺得學到的東西比較重要，我們上課的週數已經比高中生少了，真正的學習應該是上課的認真加上課後的努力！

**Comment by EPHMR0524** [comment]
我覺得中央大多數工科的教授都還是選擇18周，畢竟課程內容18周都不一定上的完了，何況是要在16周解決，而且我覺得比起很趕的把課程上完囫圇吞棗，還不如放慢速度紮實的把該學會的內容學完，這樣對未來工作以及研究所的學習都比較好

**Comment by jylin34** [comment]
我也是傾向依照課程loading分配，比較重的就18周，比較輕的就16周，這樣一來，考試也不會都集中在一起。看到有些台大的同學，課程變成16周後，上課老師講的內容變少，考試範圍仍然不變，每天都在趕進度。
